build.sbt

```diff
@@ -37,27 +38,24 @@ pomExtra := (
 licenses := Seq("Apache V2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html"))
 
 libraryDependencies ++= Seq(
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
-  "ch.qos.logback" % "logback-classic" % "1.1.9" % "test",
-  "org.specs2" %% "specs2-core" % "4.10.0" % "test"
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+  "ch.qos.logback" % "logback-classic" % "1.4.11" % "test",
+  "org.specs2" %% "specs2-core" % "4.20.3" % "test"
```

Comment:
the latest versions of specs2 are scala 3 only, so I picked the latest cross compatible version